docs/iris/example_code/General/polynomial_fit.py

```diff
@@ -0,0 +1,51 @@
+"""
+Fitting a polynomial
+====================
+
+This example demonstrates adding a polynomial fit to a 1D plot of data from
```

Comment:
This still isn't quite right, you are implying the fit is to the plot, which it is not.